config/types.go

```diff
@@ -37,3 +42,10 @@ type FTWTestOverride struct {
 	ForcePass map[string]string `koanf:"forcepass"`
 	ForceFail map[string]string `koanf:"forcefail"`
 }
+
+// FTWTestOverrideRe hold the lists transformed into regexes
+type FTWTestOverrideRe struct {
```

Comment:
koanf seems to support `TextUnmarshaler` so I believe you can avoid needing a second field / struct which makes the configuration a bit harder to reason about

Something like

```go

type FTWTestOverride struct {
  Ignore map[string]FTWRegexp
}

type FTWRegexp regexp.Regexp

func (r *FTWRegexp) UnmarshalText(b []byte) error {
  re, err := regexp.Compile(string(b))
  if err != nil {
    return fmt.Errorf("invalid regexp: %w", err)
  }
  *r = re
  return nil
}
```

Reply:
Implemented, thank you! I had to update Koanf and the resulting map is `map[*FTWRegexp]string`. If we want to keep the excluded id - comment map, the key was the id (now regex) and the string value is the comment.

Reply:
Oh. Yes, you're absolutely right. I don't know what I was thinking...